demo/azurite/docker-compose.yml

```diff
@@ -0,0 +1,113 @@
+version: '3.8'
+
+volumes:
+  pgdata:
+    driver: local
+  blobdata:
+    driver: local
+
+services:
+  postgres:
+    image: postgres:14-alpine
+    # https://dba.stackexchange.com/q/275378/46472
+    shm_size: 256MB
+    volumes:
+      - type: volume
+        source: pgdata
+        target: /var/lib/postgresql/data/
+      - type: bind
+        source: ./postgres/init
+        target: /docker-entrypoint-initdb.d
+    environment:
+      POSTGRES_DB: fhirdb
+      POSTGRES_USER: fhiradmin
+      POSTGRES_PASSWORD: change-password
+      PGDATA: /var/lib/postgresql/data/pgdata
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -d fhirdb -U fhiradmin"]
+      start_period: 22s
+      interval: 5s
+      timeout: 4s
+      retries: 3
+    # "Larger settings for shared_buffers usually require a corresponding increase in max_wal_size, 
+    # to spread out the process of writing large quantities of new or changed data over a longer period of time."
+    command: "
+      -c shared_buffers=2GB
+      -c max_wal_size=3GB
+      -c max_connections=100
+      -c max_prepared_transactions=100
+      -c max_locks_per_transaction=128
+      "
+    ports:
+      - 5432:5432
+    networks:
+      - fhir
+
+  azurite:
+    image: mcr.microsoft.com/azure-storage/azurite
+    volumes:
+      - type: volume
+        source: blobdata
+        target: /data
+    ports:
+      - 10000:10000  # Azurite Blob service
+    networks:
+      - fhir
+
+  fhir-server:
+    image: linuxforhealth/fhir-server:latest
+    hostname: fhir
+    volumes:
+      - type: bind
+        source: ./fhir/config
+        target: /opt/ol/wlp/usr/servers/defaultServer/config
+        read_only: true
+      - type: bind
+        source: ./fhir/configDropins/overrides
+        target: /opt/ol/wlp/usr/servers/defaultServer/configDropins/overrides
+        read_only: true
+      - type: bind
+        source: ./fhir/userlib
+        target: /opt/ol/wlp/usr/servers/defaultServer/userlib
+        read_only: true
+      - type: bind
+        source: ./fhir/openapi.json
+        target: /opt/ol/wlp/usr/servers/defaultServer/apps/expanded/fhir-openapi.war/META-INF/openapi.json
+        read_only: true
+    environment:
+        TRACE_SPEC: "com.ibm.fhir.*=FINE"
```

Comment:
Perhaps INFO would make a  better default.

Reply:
I thought it might be nice to have the FINE logging while demoing so we can show all the work that is happening, but I suppose you are right that less output would be the better default.  Updated.